recipes/sip/bld.bat

```diff
@@ -0,0 +1,8 @@
+%PYTHON% configure.py --sysroot=%PREFIX% --bindir=%LIBRARY_BIN%
```

Comment:
I don't think so because `sip` only installs one header file (called `sip.h`) in %PREFIX%\include, which is the location used for other Python libraries, and not for C/C++ ones.

Besides, our PyQt recipe builds correctly on top of this one :-)